dlt/destinations/dataset/dataset.py

```diff
@@ -171,6 +169,40 @@ def row_counts(
         # Execute query and build result dict
         return self(query)
 
+    def list_load_ids(
+        self, status: Union[int, list[int], None] = 0, limit: Union[int, None] = 10
+    ) -> SupportsReadableRelation:
```

Comment:
this should be implemented as ibis expression and raise if ibis is not installed